pkg/controllers/management/etcdbackup/etcdbackup.go

```diff
@@ -322,7 +322,8 @@ func GetS3Client(sbc *v3.S3BackupConfig, timeout int) (*minio.Client, error) {
 	}
 	var s3Client = &minio.Client{}
 	var creds *credentials.Credentials
-	var tr = http.DefaultTransport
+	var tr http.RoundTripper = &http.Transport{}
+	copier.Copy(tr, http.DefaultTransport)
```

Comment:
Presumably this would copy pointers over such that the old transport and the new transport would point to the same underlying objects. It looks like that means this `tr` would share DefaultTransport's DialContext. Could that cause a future problem if we then modify the DialContext on `tr`?

Reply:
@cjellick we do modify it actually on line 327. Just to be safe, I'll better initialize the struct instead of copying it.